src/Directory.Packages.props

```diff
@@ -32,6 +33,7 @@
     <PackageVersion Include="Npgsql" Version="7.0.1" />
     <PackageVersion Include="Polly" Version="7.2.3" />
     <PackageVersion Include="StyleCop.Analyzers" Version="1.1.118" />
+    <PackageVersion Include="Swashbuckle.AspNetCore.SwaggerUI" Version="6.5.0" />
```

Comment:
Did you happen to check the autogenerated NOTICE.txt in your build if it contains attribution to SwaggerUI?

Reply:
Our pipeline was only generating notice.txt for release builds. Updated file to always create notice file to see what is included. SwashBuckle.SwaggerUI is MIT, but it does include the base SwaggerUI package which is Apache. They don't seem to show an explicit notice, so i may have to update my script which manually adds the license notice in. Running a build now to see what notice file includes.

Reply:
only `Swashbuckle.AspNetCore.SwaggerUI 6.5.0 - MIT` was added to the notice file.

Reply:
is the base dependency `SwaggerUI` also required to be added which is `Apache`? What do we do for other components?

Reply:
No, SwaggerUI is not explicitly included. Only Swashbuckle.AspNetCore.SwaggerUI. We don't have a mechanism which recursively checks dependencies of dependencies to see whether additional licensing information needs to be included in a notice file. We are dependent on the license used in direct dependencies.